app/workers/external_services_status_job.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class ExternalServicesStatusJob
+  include Sidekiq::Worker
```

Comment:
If it runs once a minute do we want it to retry? Could set retry false or maybe one time?

Reply:
Yeah, I was thinking the same thing.  Good call.  I'll set it to retry false, b/c it is going to run 60 seconds later anyway.